serialization/journal.proto

```diff
@@ -455,18 +450,18 @@ message DeserializePlugin {
     optional DeserializePlugin extension = 5050;
   }
   message In {
-    required string serialization = 1 [(size) = "serialization_size"];
+    required string serialization = 1;
     required fixed64 deserializer = 2
         [(pointer_to) = "PushDeserializer",
-         (is_consumed_if) = "serialization->empty()"];
+         (is_consumed_if) = "*serialization == '\\0'"];
```

Comment:
`std::string_view(serialization).empty()`?

Reply:
(We might want to change the API so that termination is signalled by `serialization == nullptr` but that is a job for another pull request.)

Reply:
Done, but note how this is less efficient (not that we care, I think it's only used when playing a journal).